### PR TITLE
allow for users from different sites to exist

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: gem/backends.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class GemModelBackend(MoloModelBackend):
+
+    def authenticate(self, request, username=None, password=None, **kwargs):
+        if username is None:
+            username = kwargs.get(UserModel.USERNAME_FIELD)
+        try:
+            user = UserModel.objects.get(
+                gem_profile__migrated_username=username,
```
Comment (miltontony) [inline gem/backends.py]: This allows a user to login using the original username.

**Comment by miltontony** [comment]
@codiebeulaine @saeedmb8 ready for review

**Comment by codiebeulaine** [comment]
👍 👍 👍

**Comment by saeedmb8** [comment]
:sparkles::dizzy:                  :sparkles::dizzy:
  :sparkles: :star: :star: :dizzy:    :sparkles: :star: :star::dizzy:
:sparkles::star::cake::cake::cake::star::cake::cake::cake::star::dizzy:
:sparkles::star::cake::cake::cake::cake::cake::cake::cake::star::sparkles:
:dizzy::star::cake::cake::cake::cake::cake::cake::cake::star::sparkles:
   :dizzy::star::cake::cake::cake::cake::cake::star::sparkles:
         :dizzy::star::cake::cake::cake::star::sparkles:
               :dizzy::star::cake::star::sparkles:
                     :dizzy::star::sparkles:

**Pull request merged**